cirq/study/sweepable.py

```diff
@@ -55,7 +55,24 @@ def to_sweeps(sweepable: Sweepable) -> List[Sweep]:
     if isinstance(sweepable, Sweep):
         return [sweepable]
     if isinstance(sweepable, dict):
-        return [_resolver_to_sweep(ParamResolver(cast(Dict, sweepable)))]
+        expandsweepable = [cast(dict, {})]
+        for key in sweepable.keys():
```

Comment:
You should be able to use something like:

`for key,value in sweepable.items():`